apps/music/css/view.css

```diff
@@ -15,65 +15,14 @@ body {
   -moz-user-select: none;
 }
 
-a {
-  text-decoration: none;
```

Comment:
No visible regressions on home view, but added the property to `.tile` just in case :)